apps/api/v2/src/modules/slots/slots-2024-09-04/services/slots.service.ts

```diff
@@ -147,13 +148,30 @@ export class SlotsService_2024_09_04 {
     }
 
     const nonSeatedEventAlreadyBooked = !eventType.seatsPerTimeSlot && booking;
-    if (nonSeatedEventAlreadyBooked) {
+    const isRoundRobinEvent = eventType.schedulingType === SchedulingType.ROUND_ROBIN;
+
+    if (nonSeatedEventAlreadyBooked && !isRoundRobinEvent) {
       throw new UnprocessableEntityException(`Can't reserve a slot if the event is already booked.`);
     }
 
-    const reservationDuration = input.reservationDuration ?? DEFAULT_RESERVATION_DURATION;
+    if (isRoundRobinEvent) {
+      const totalSeats = eventType.hosts.filter((host) => host.isFixed === false).length;
```

Comment:
I don't understand this logic. If we have a round robin event with 1 fixed host and 2 not fixed, then by this logic we can reserve it 2 times, but how is that possible if a fixed host can be only in a meeting once, right?

I went on web app and created round robin with 1 fixed and 2 random guests and booked at 10AM and then opened event in incognito and the 10AM slot is gone.

Reply:
but fixed host can only be in 1 meeting at 9AM slot